neqo-http3/src/features/extended_connect/webtransport_streams.rs

```diff
@@ -199,4 +213,18 @@ impl SendStream for WebTransportSendStream {
         }
         Ok(())
     }
+
+    fn stats(&mut self, conn: &mut Connection) -> Res<SendStreamStats> {
+        let mut acked = conn.stream_bytes_acked(self.stream_id)?;
+        acked -= self.bytes_non_app_data;
+        let stats = SendStreamStats::new(
+            SystemTime::now()
+                .duration_since(UNIX_EPOCH)
+                .expect("SystemTime before UNIX EPOCH!"),
```

Comment:
We should not be invoking `SystemTime::now()` in this code (or `Instant::now()`).  We always have the caller tell us what time it is.

Reply:
OK, so maybe we don't need to have `timestamp` in `SendStreamStats` here.